style.css

```diff
@@ -0,0 +1,153 @@
+
+body {
+    margin: 0;
+    height: 100vh;
+    width: 100vw;
+    background-color: black;
+    background-size: 100%;
+    background-repeat: no-repeat;
+}
+/* *************************************************************** */
+                /*  all items in header below */
+/* *************************************************************** */
+header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 15%;
+    font-size: 2rem;
+    color: red;
+    font-family: 'Russo One', sans-serif;
+}
+
+h2{
+    height: 6vh;
+    font-size: 10vmin;
```

Comment:
I would rather see the base font for everything be dynamic and the part here a percentage on font.